CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changes to the Mapbox Navigation SDK for iOS
 
+## v2.10.0 [UNRELEASED]
```

Comment:
Hopefully this “unreleased” suffix doesn’t break the release automation script. I think we used to say “main” to avoid tripping any regular expression looking for a version number.